stl/inc/algorithm

```diff
@@ -3436,6 +3436,19 @@ _CONSTEXPR20 _FwdIt2 swap_ranges(const _FwdIt1 _First1, const _FwdIt1 _Last1, _F
     const auto _UFirst1 = _Get_unwrapped(_First1);
     const auto _ULast1  = _Get_unwrapped(_Last1);
     const auto _UFirst2 = _Get_unwrapped_n(_First2, _Idl_distance<_FwdIt1>(_UFirst1, _ULast1));
+
+#if _ITERATOR_DEBUG_LEVEL == 2 && _HAS_IF_CONSTEXPR

```

Comment:
Guess `_HAS_IF_CONSTEXPR` is no longer needed?

Reply:
Correct! I've pushed a merge, followed by a commit to drop `&& _HAS_IF_CONSTEXPR`.